src/publish/confluence/api/index.ts

```diff
@@ -381,10 +381,19 @@ export class ConfluenceClient {
     };
   }
 
-  private handleResponse<T>(response: Response) {
+  private async handleResponse<T>(response: Response) {
```

Comment:
In case anyone else looks at this: the change to async here changes the type of `handleResponse`. That means the handleResponse call sites change as well, but inspecting the code here shows that the only two call sites are already async, and call handleResponse in tail position. That means those calls do not need an additional `await`.